hardware/battery/battery_management_system_protocol.ksy

```diff
@@ -0,0 +1,223 @@
+meta:
+  id: battery_management_system_protocol
+  title: Communication protocol of smart battery management systems from LLT power
+  license: CC0-1.0
+  ks-version: 0.8
+  endian: be
```

Comment:
`bit-endian: be`

Reply:
Why set minimum version to 0.9 if it is tested and working with 0.8?

Reply:
@Jakeler:

> Why set minimum version to 0.9 if it is tested and working with 0.8?

See https://doc.kaitai.io/user_guide.html#bit-endian:

<table role="table">
<tbody>
<tr>
<th>IMPORTANT</th>
<td><p>Support for <code>meta/bit-endian</code> was added in <strong>v0.9</strong> (previously only BE direction was supported). To maintain backward compatibility, the big-endian order (<code>be</code>) is default.</p><div>However, if you don’t really need to support pre-0.9 KSC versions, it’s recommended to state <code>meta/bit-endian: be</code> explicitly, because it raises awareness about the existence of LE bit endianness.</div></td>
</td>
</tbody>
</table>

Reply:
Okay, changed in 87ee4a0